src/popup/locales/en.json

```diff
@@ -578,6 +579,7 @@
     "token-details": {
       "details": "Token Details",
       "send": "Send",
+      "propose-tx": "Propose Tx",
```

Comment:
I suggest to use camelCase as we are moving to this naming convention within the translation files.